### ISSUE TITLE
Error Syncing with Skype

### ISSUE THREAD

**Issue by clienthax** [suurjaak/Skyperious#106]
V4.8.1

Syncing a DB made of a few merged ones for the same account

```
Synchronizing messages..

Error syncing from Skype online service:

Traceback (most recent call last):
  File "workers.py", line 839, in run
  File "live.py", line 715, in populate
  File "live.py", line 794, in populate_history
  File "live.py", line 193, in build_msg_cache
  File "skypedata.py", line 362, in execute
OperationalError: no such column: timestamp__ms
```

**Comment by clienthax** [comment]
<img width="1136" alt="image" src="https://user-images.githubusercontent.com/1056270/159054962-25cc168f-ed8e-457d-9a23-6b1d056fa193.png">
Am I doing something wrong here?

**Comment by clienthax** [comment]
Aha looks like the : should have been a ; on the end of the first line, seems to be running the sync now

**Comment by suurjaak** [comment]
Yep, sorry, the instruction I gave had a typo.

**Comment by suurjaak** [comment]
With the newly released version 5.0, Skyperious will now also auto-update the schema of an older format database on live sync.
